magenta/models/polyphony_rnn/README.md

```diff
@@ -2,8 +2,17 @@
 
 This model applies language modeling to polyphonic music generation using an LSTM. Unlike melodies, this model needs to be capable of modeling multiple simultaneous notes. Taking inspiration from [BachBot](http://bachbot.com/), we model polyphony as a single stream of note events with special START, STEP_END, and END symbols. Within a step, notes are sorted by pitch in descending order.
 
-For example, using the default quantizing resolution of 4 steps per quarte note, a sequence containing only a C Major chord with a duration of one quarter note would look like this:
+The rhythmic distance of every step composes a 1/16th note. For example, *assuming the default quantizing resolution of 4 steps per quarter note*, notating a C Major chord with a duration of 1/16th note would be as follows:
```

Comment:
I think this may be more clear if the first sentence is removed because the rhythmic distance of each step depends on the quantizing resolution.